Start36 (#3330)
modelica/ModelicaSpecification: chapters/inheritance.tex

\subsection{Removing Modifiers -- break}\label{removing-modifiers-break}

> Modifications may contain the special keyword \lstinline!break! instead of an expression.
> \index{break@\robustinline{break}}

Reviewer: We tend to have the `\index` next to the term in the source, and make use of dedicated macro `\indexinline`:
```suggestion
Modifications may contain the special keyword \lstinline!break!\indexinline{break} instead of an expression.
```

Reviewer: With more than one entry for `break`, it would also be good to explain how they differ in the index, but then the dedicated macro must be given up again:
```suggestion
Modifications may contain the special keyword \lstinline!break!\index{break@\robustinline{break}!removing modifier} instead of an expression.
```

parameter Boolean useHeatPort = false annotation(choices(checkBox = true));
\end{lstlisting}
\end{example}

> \section{Selective Model Extension}\label{selective-model-extension}\index{break@\robustinline{break}!deselection}\index{deselection}

Reviewer: Two things.  First explain in the document index that the _deselection_ entry is really meant to bring you to a section that has nothing about deselection in the title:
```suggestion
\section{Selective Model Extension}\label{selective-model-extension}\index{deselection!selective model extension}
```

Second, move the other index entry to where the use of the keyword `break` is introduced further down in this section:
```
\begin{lstlisting}[language=modelica]
model C
  extends B($\ldots$, break $D$, $\ldots$);
  $\ldots$
end C;
\end{lstlisting}%
\index{break@\robustinline{break}!deselection}
```

\end{lstlisting}
\end{example}

> \section{Selective Model Extension}\label{selective-model-extension}
> \index{break@\robustinline{break}}\index{deselection}

Reviewer: Qualify index entry, and customary empty line for source code readability:
```suggestion

\section{Selective Model Extension}\label{selective-model-extension}\index{break@\robustinline{break}!deselection}\index{deselection}
```

Reviewer: I would also have preferred to have the index entry for `break` located closer to the first mention of `break` in the section.  It makes it easier to quickly find out the role of `break` in this context, which helps determine if this is the entry one is really interested in.
